src/Websites/Websites/Utilities/WebsitesClient.cs

```diff
@@ -112,9 +112,11 @@ public void UpdateWebApp(string resourceGroupName, string location, string webAp
 
             if (siteEnvelope != null)
             {
-                System.Reflection.PropertyInfo property = siteEnvelope.GetType().GetProperty("VnetInfo");
-                if (property != null)
-                    property.SetValue(siteEnvelope, null);
+                if (siteEnvelope is PSSite)
+                {
+                    PSSite site = siteEnvelope as PSSite;
```

Comment:
No need to use `as` operator here since the siteEnvelope is for sure an instance of PSSite
`((PSSite)siteEnvelope).VnetInfo = null;` works.

Reply:
Done!